src/components/dialogs/VerifyApiDialog/utils.ts

```
@@ -1,20 +1 @@
- * Stringify an unknown value for display in the UI.
- * Falls back to a best-effort string when the value is not JSON-serializable.
- */
-export function safeJsonStringify(value: unknown): string {
-  if (value === undefined) return ""
-  try {
-    return JSON.stringify(value, null, 2)
-  } catch {
-    return String(value)
-  }
-}
+export { formatLatency, safeJsonStringify } from "../shared/utils"
```

ESLint

[error] Unable to resolve path to module '../shared/utils'.

(import/no-unresolved)